migrations/0000_createDbs.sql

```diff
@@ -1,26 +1,31 @@
 -- Types
-CREATE TYPE note_type AS ENUM ('predictions', 'marginalia', 'meta', 'questions');
+CREATE TYPE note_type AS ENUM ('uncategorized', 'predictions', 'marginalia', 'meta', 'questions');
 
 -- Tables
-CREATE TABLE IF NOT EXISTS users(
+CREATE TABLE IF NOT EXISTS user(
 	id bigserial PRIMARY KEY,
 	display_name text NOT NULL,
 	email_address text UNIQUE NOT NULL,
 	password bytea NOT NULL,
 	creation_time timestamp NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS publications(
+CREATE TABLE IF NOT EXISTS publication(
 	id bigserial PRIMARY KEY,
-	author_id bigint references users(id) NOT NULL,
+	author_id bigint references user(id) NOT NULL,
 	creation_time timestamp NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS notes(
+CREATE TABLE IF NOT EXISTS note(
 	id bigserial PRIMARY KEY,
-	author_id bigint references users(id) NOT NULL,
+	author_id bigint references user(id) NOT NULL,
 	type note_type,
 	content text NOT NULL,
-	publication_id bigint references publications(id),
 	creation_time timestamp NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS publication_to_note_linkage(
```

Comment:
`linkage` sounds weird to me, but i'm pretty sure it's not wrong. alternatives would be appreciated, but unnecessary

Reply:
honestly i'm not a fan of this table name either. I just couldn't think of any better alternatives. ptn_reference seems to vague, ptn_association also vague. Maybe ptn_relationship?